x-pack/plugins/profiling/server/routes/flamechart.ts

```diff
@@ -73,17 +71,12 @@ export function registerFlameChartSearchRoute({
             totalFrames,
             samplingRate
           );
-          logger.info(`creating callee tree took ${Date.now() - t1} ms`);
 
-          const t2 = Date.now();
           const fg = createBaseFlameGraph(tree, samplingRate, totalSeconds);
-          logger.info(`creating flamegraph took ${Date.now() - t2} ms`);
 
           return fg;
         });
 
-        logger.info('returning payload response to client');
-
         return response.ok({ body: flamegraph });
       } catch (error) {
         return handleRouteHandlerError({ error, logger, response });
```

Comment:
__issue:__ I'm concerned that `handleRouteHandlerError` returns the actual error message to the client, which could easily leak sensitive information. Since `try/catch` blocks are large and capture various functionality, the risk might be high here. Can you just log the error message, but return something less detailed (e.g. `handleRouteHandlerError({ error, logger, response, message: 'Failed to ....' })`)?